src/common/utils.js

```diff
@@ -33,12 +31,12 @@ export const getCurrentLocale = () => {
     const language = i18n.language;
     switch (language) {
         case "nb":
-            return "no-NO";
+            return "nb-NO";
         case "nn":
-            return "no-NO";
+            return "nb-NO";
```

Comment:
Er det riktig?

Reply:
Ja, dvs - "nn-No" støttes ikke av Safari, mens "nb-NO" fungerer likt på Chrome og Safari. Antar at format for norsk er likt uavhengig av om det er nynorsk eller bokmål

Reply:
Så vi kan egentlig bare endre dette til 
`case "nb"
case "nn"
 return "nb-NO"; `